src/game.c

```diff
@@ -203,6 +207,7 @@ int computer_take_turn(void)
     else
     {   /*If there is playable card, then remove the first playable card from on hand cards list*/
         Deck_t* discard_card = remove_first_playable_card(&g_players[COMPUTER].cards_on_hand);
+        printf("Computer Drops..(%s, %s)\n", CARD_COLOR_STRING[discard_card->card.color], CARD_NAME_STRING[discard_card->card.name]);
         memcpy(&g_card_on_table, &discard_card->card, sizeof(Card_t));
         //printf("Card on the table now: (%s, %s)\n", CARD_COLOR_STRING[discard_card->card.color], CARD_NAME_STRING[discard_card->card.name]);
```

Comment:
Let's delete this printf.

Reply:
This printf is needed to make the game interactive. Personally, I felt lost having to compare the card on table to its previous value to check if the computer has dropped a card or not. I feel this make things more clear

Reply:
I mean this one: //printf("Card on the table now: (%s, %s)\n", CARD_COLOR_STRING[discard_card->card.color], CARD_NAME_STRING[discard_card->card.name]);

Reply:
Got it.